doc/docs/Python_User_Interface.md

```diff
@@ -1081,7 +1081,7 @@ Once you have called `add_flux`, the Fourier transforms of the fields are accumu
 <div class="class_members" markdown="1">
 
 ```python
-def display_fluxes(self, *fluxes): 
+def display_fluxes(self):
```

Comment:
The function `display_fluxes` has an input parameter `*fluxes` that is described in the documentation which should therefore not be removed.